content/en/docs/chart_template_guide/function_list.md

```diff
@@ -1562,6 +1562,13 @@ $copy := compact $list
 `compact` panics if there is a problem and `mustCompact` returns an error to the
 template engine if there is a problem.
 
+### index
+
+To get the nth element of a list, use `index list [n]`. To index into multi-dimensional lists, use 
```

Comment:
my apologies. The document is hard-wrapped at 80 characters. Can you please amend your commit?

Reply:
gotcha, amended now